Roles/Director/README.md

```diff
@@ -0,0 +1,9 @@
+# Director
+
+1. The Director [role](..) is responsible for representing the customer/end-user/stakeholders for a given project or task.
```

Comment:
Should we reference the ICG Managing Director here or defer?

Reply:
If you think it's necessary, say that the Director is held by the ICG managing director. And that the treasurer is held by the Ulex Advisory board.

Reply:
OK done